src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -171,6 +171,7 @@ their names (or descriptions) depend on some threshold.  Those filters are not i
     // Symbolic alleles
     public final static String SYMBOLIC_ALLELE_DEFINITION_HEADER_TAG = "ALT";
     public final static String NON_REF_SYMBOLIC_ALLELE_NAME = "NON_REF";
```

Comment:
Could we replace uses of this with uses of `Allele.NON_REF_STRING`.

Reply:
Done.

Reply:
Wait, actually, we can't. `Allele.NON_REF_STRING` is the string for the symbolic allele ("<NON_REF>", whereas this is just the `NON_REF`. Allele doesn't seem to have just the string.